src/test/java/seedu/address/testutil/TypicalVideos.java

```diff
@@ -8,7 +8,7 @@
 public class TypicalVideos {
 
     public static final Video CONTENT_VIDEO = new VideoBuilder().withName("Vid1")
-            .withHasWatched(true).build();
+            .withHasWatched(true).withTags("Content").build();
```

Comment:
Don't think this should be done because "Vid1" is meant to represent a tag-less video. It's intentionally made to not have tags for testing stuff like serializing and de-serializing tagless items